### DISCUSSION TITLE
这周你做了什么？2022-05-20

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#19] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by jiacai2050** [comment]
- 啥都没干，主要是开始寻找下一期嘉宾了，候选人员 youmu、第253页图灵笔记。。。
- 今天 520 是时候表白一下了

![101_300x300_Front_Color-BlackWhite](https://user-images.githubusercontent.com/3848910/169438089-4abd8d7e-6214-4d4a-af74-c1c6e4e27078.jpeg)

**Comment by yuzhou721** [comment]
- lsp-bridge 在doom配置下要怎么弄呀？能给点思路不？
- 520要花钱买花了

**Reply by jiacai2050** [reply]
你了解 load-path 的作用吗？了解的话应该不难配置，不了解的话可以参考
- https://liujiacai.net/blog/2021/05/05/emacs-package/

**Reply by yuzhou721** [reply]
主要是怕和doom在用着的company冲突

**Comment by calsys456** [comment]
论文写完了，巨菜[捂脸]  http://rosa.sdf.org/#org00adc5d

然后用word排版给我气得够呛，macos自带的C-fbpn移动都被它给拦下来了，难受，于是就开始动手写一个emacs的富文本编辑功能，现在基础功能都写好了，正在看docx格式，还写了个拖动posframe panel的功能，算是给上周还愿了吧哈哈

不知道这个大家会不会喜欢，毕竟有enhanced-mode了，所以也不太敢上坛子发帖🙈 但enhanced-mode真的能用嘛，感觉它功能好有限🤣

**Reply by jiacai2050** [reply]
抽空拜读一下大作，哈哈。

**Reply by jiacai2050** [reply]
enhanced-mode 是指哪个？链接贴一下吧

**Reply by qiandu2006** [reply]
> 3. 一、 Emacs 的诞生

一点小瑕疵，这个格式逼死强迫症～～～

**Reply by calsys456** [reply]
> enhanced-mode 是指哪个？链接贴一下吧

噗，丢人丢大惹，是enriched-mode……我滚回去补习英语哈哈哈哈😂

**Reply by calsys456** [reply]
> > 5\. 一、 Emacs 的诞生
> 
> 一点小瑕疵，这个格式逼死强迫症～～～

哈哈哈哈谢谢你的阅读，我也差点没把我自己逼死😭

**Comment by suckerSlayer** [comment]
开始入坑emacs的一周...断断续续开着教程做背景音，配置了Evil，踩了终端下zsh在emacs的M-x shell里无法工作的坑。另有人知道这个shell里的prompt颜色为何无法被iTerm的主题色影响么

**Reply by jiacai2050** [reply]
不建议用 TUI，后面还有各种问题，最烦的是按键冲突，建议直接 GUI 吧
